data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,129 @@ mission "Strider notification"
+			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you! Is <payment> okay?" This last part is said in almost a whisper.`
+			choice
+				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
+					goto question
+				`	"Sure, I'm not hurting for cash right now."`
+					goto accept
+				`	"No way, that's too low. Find someone else."`
+						decline
+			label question
+			`	"Well, that's kind of why we want to go there," says one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
+			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`
+			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`
+			choice
+				`	"Is this legal?"`
+				`	"Not my business where you spend your vacations. Come aboard."`
+					goto accept
+			`	The students look at each other, considering this question for what is apparently the first time. "We've heard of other people going there with no problems, as long as they were... discreet about it," says one of the students, trying to assure both you and their fellow classmates. Another mentions having heard something about needing a permit to enter, but he is quickly shushed by the rest of the group.`
+			choice
+				`	"I'm not about to risk getting in trouble for this. Find someone else."`
+						decline
+				`	"Not my business where you spend your vacations. Welcome aboard."`
+			label accept
+			`	The students race onboard your ship. As they settle into their bunks, you can hear them speculating about what they will find on <planet>.`
+				accept
+	on visit
+		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
+	on complete
+		payment 8000
+		dialog `Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>. After passing the time in your ship for a few hours, you are shocked by a loudspeaker sternly pronouncing something in Hai. You figure you should head back to the spaceport to see what's going on. Then again, disassociating yourself from them may save you some trouble with law enforcement.`
+
+
+
+mission "Station Exploration 2"
+	name "Station Exploration"
+	description "Bring a group of Hai and human classmates back home to <planet>."
+	minor
+	source "Darkrest"
+	destination "Greenwater"
+	passengers 5
+	to offer
+		has "Station Exploration 1: done"
```

Comment:
```suggestion
		has "Station Exploration 1: done"
		not "Station Exploration Abandon Ship: declined"
```
So that you don't get this mission after you've been told off.